lib/api/Healthcheck.js

```diff
@@ -84,6 +94,14 @@ class Healthcheck {
                 topics[this._repConfig.topic] = topicMd;
                 connections.isrHealth = this._checkISRHealth(topicMd);
             }
+            if (topicMd && this._isMissingISR(topicMd)) {
+                const error = {
+                    method: 'Healthcheck.getHealthcheck',
+                    error: 'missing in-sync replicas',
```

Comment:
The error message could benefit to be rephrased, as it can be confused as actually missing *some* in-sync replicas, while it shall mean at least one partition has *no* in-sync replica available.

Reply:
Agree. I've updated the message to 'partition missing in-sync replica'.